libpod/container_api.go

```diff
@@ -840,7 +840,7 @@ func (c *Container) ShouldRestart(ctx context.Context) bool {
 
 // CopyFromArchive copies the contents from the specified tarStream to path
 // *inside* the container.
-func (c *Container) CopyFromArchive(ctx context.Context, containerPath string, tarStream io.Reader) (func() error, error) {
+func (c *Container) CopyFromArchive(ctx context.Context, containerPath string, chown bool, tarStream io.Reader) (func() error, error) {
```

Comment:
Lets do this with a options structure, So that when we open a PR for links, we don't have to change the interface again.

```
struct CopyArchiveOptions {
           Chown bool,
}
```